Google.Ads.GoogleAds/examples/Recommendations/DetectAndApplyRecommendations.cs

```diff
@@ -131,6 +132,7 @@ public void Run(GoogleAdsClient client, long customerId)
                                     "{keyword.MatchType}");
                             }
 
+                            // [START build_apply_recommendation_operation]
```

Comment:
It looks like this piece is missing some of the comments from the other libs' examples (about how you would override recommended values), and it isn't in its own separate function like the updated example. @Raibaz could you update this to align with the latest changes, so what's shown in devsite is the same across libs?

Reply:
The separate function piece isn't very important so that can be left in, but could you update the comments? @danielfrg Might be easiest for you to make the update but loop in Mattia for the code.

Reply:
I updated the example to reflect the most recent changes in the PHP example, should be ok now.